objectRecognitionSource/src/main/java/io/vantiq/extsrc/objectRecognition/ObjectRecognitionCore.java

```diff
@@ -304,25 +306,35 @@ public void sendDataFromImage(ImageRetrieverResults imageResults) {
             return;
         }
         
+        Map<String, Object> processingParams = new LinkedHashMap<>();
+        processingParams.put("imageTimestamp", imageResults.getTimestamp());
+        
         // Send the results of the neural net if it doesn't error out
+        NeuralNetInterface localNeuralNet = null;
         try {
-            synchronized (this) {
+            synchronized (SYNCH_KEY) {
```

Comment:
are we purposely changing from synchronizing only within this instance to across all instances?  The SYNC_KEY isn't really adding much.  The notion of the named string or object is useful if there are multiple independent entities within, say, a class that need to be independently synchronized.  In that case, sync on the class or instance will work, but it's overly confining.